Code/RDGeneral/StreamOps.h

```diff
@@ -145,28 +145,56 @@ inline boost::uint32_t readPackedIntFromStream(std::stringstream &ss) {
   int shift, offset;
   char tmp;
   ss.read(&tmp, sizeof(tmp));
+  if (ss.fail()) {
```

Comment:
Perhaps we could roll this into an ss_check inline function.  We might want to check for premature eof as well.

Reply:
I thought about that, but a premature EOF in the middle of reading the char here should result in the failbit being set (since the requested number of bytes wasn't read), so that case is covered. And if we hit EOF after reading the bytes we don't care (at least not here).